src/scaffoldmaker/meshtypes/meshtype_3d_lung1.py

```diff
@@ -780,3 +684,496 @@ def refineMesh(cls, meshrefinement, options):
         refineElementsCount = options['Refine number of elements']
         meshrefinement.refineAllElementsCubeStandard3d(refineElementsCount, refineElementsCount, refineElementsCount)
 
+def getLungNodes(lungSide, cache, coordinates, generateParameters, nodes, nodetemplate, nodeFieldParameters,
+                 lElementsCount1, lElementsCount2, lElementsCount3,
+                 uElementsCount1, uElementsCount2, uElementsCount3,
```

Comment:
Can it work with any values?
I'm not asking you to change it, but generally don't pass in values that have to be fixed (unless asserted).
I suggest you document where the counts were assigned that these are the only values that work.

Reply:
I will add a comment above these parameters.